cloudos/__main__.py

```diff
@@ -15,18 +15,28 @@
 
 
 @click.group()
+@click.version_option()
 def run_cloudos_cli():
     """CloudOS python package: a package for interacting with CloudOS."""
     print(run_cloudos_cli.__doc__ + '\n')
 
 
 @run_cloudos_cli.group()
+@click.version_option()
 def job():
     """CloudOS job functionality: run and check jobs in CloudOS."""
     print(job.__doc__ + '\n')
 
 
+@run_cloudos_cli.group()
+@click.version_option()
```

Comment:
For the `cloudos <module> --version` option, I am not sure if it adds extra value, beyond the `cloudos --version`.
What is the most important for the user for `cloudos <module>` is `cloudos <module> --help`.
Also, since we have all the modules in a monorepo of sorts, regardless of what module changes, all modules get the version bump. Eg, between 1.0.0 and 1.1.0 even if `jobs` is unaltered, and the only change is the addition of `workflows`, all the repo gets bumped, so the concept of module specific versioning is superseeded.

Is it worth adding to the help the version as well, and removing `--version` from specific modules, eg `jobs`, `workflows`, and keep it only in the package level ie `cloudos --version`?

@dapineyro @sk-sahu